src/main/kotlin/uk/gov/justice/digital/hmpps/prisontonhs/services/PrisonerChangeListenerPusher.kt

```diff
@@ -9,25 +9,26 @@ import org.springframework.stereotype.Service
 
 
 @Service
-open class PrisonerMovementListenerPusher(
-    private val prisonMovementService: PrisonMovementService
+open class PrisonerChangeListenerPusher(
+    private val prisonerPatientUpdateService: PrisonerPatientUpdateService
 ) {
   companion object {
     val log: Logger = LoggerFactory.getLogger(this::class.java)
     val gson: Gson = GsonBuilder().create()
   }
 
   @JmsListener(destination = "\${sqs.queue.name}")
-  open fun pushPrisonUpdateToProbation(requestJson: String?) {
+  open fun pushPrisonUpdateToxProbation(requestJson: String?) {
```

Comment:
Is the 'x' in the method name intentional?

Reply:
typo!